src/components/Owners/Providerscard.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import PropTypes from 'prop-types'; 
+// import haircutImg from "../Images/haircut.svg";
+import maleServiceProvider from "../Images/maleServiceprovider.png";
+import femaleServiceProvider from "../Images/femaleServiceproviders.jpg";
+
+
+function Providerscard({providersProperties}) {
+  const providerName = providersProperties.name;
+  const image = providersProperties.image;
+  const gender = providersProperties.gender;
+  
+  return (
+    <div id='providersCardContainer'>
+
+      <div id='providersCardimgContainer'>
+        {image?(
+          <img src={image} alt=""  />
+        ):(
+           gender==='male'?(
```

Comment:
Looks like a typo , I think it should ==

Reply:
actually in javascript there is === which compares strictly based on data type and value if i use == then it will only check the value and not the data type so it said better practice to use ===

Reply:
Thanks, good to know